exams/pearson.py

```diff
@@ -0,0 +1,147 @@
+"""
+Pearson specific exam code
+"""
+from functools import partial
+from operator import attrgetter
+import csv
+import logging
+
+from django.conf import settings
+import pycountry
+import pysftp
+
+from exams.exceptions import (
+    InvalidProfileDataException,
+    InvalidTsvRow,
+)
+
+# custom csv dialect for Pearson
+csv.register_dialect(
+    'pearsontsv',
+    delimiter='\t',
+)
+
+PEARSON_DATETIME_FORMAT = "%Y/%m/%d %H:%M:%S"
+
+log = logging.getLogger(__name__)
+
+
+def format_datetime(dt):
+    """
+    Formats a datetime to Pearson's required format
+    """
+    return dt.strftime(PEARSON_DATETIME_FORMAT)
+
+
+def get_field_mapper(field):
+    """
+    Returns a field mapper, accepts either a property path in str form or a callable
```

Comment:
Actually, if you only use these functions in this file feel free to just add a `_` prefix